examples/basic-tracer-node/docker/ot/docker-compose.yaml

```diff
@@ -10,7 +10,7 @@ services:
     ports:
       - "9464:9464"
       - "4317:4317"
-      - "55681:55681"
+      - "4318:4318"
```

Comment:
The collector docker image being used in this example is 0.27.0 (line 6).
In this version, the otlp port is 55681.
Default port of 4318 is introduced in v0.32.0 [here](https://github.com/open-telemetry/opentelemetry-collector/blob/main/CHANGELOG.md#-enhancements--10)

I suspect that changing the port to 4318 might break this example. We need to either upgrade the image tag to something later than 0.32.0, or set the 4318 port also in `collector-config.yaml` file for this package.